packages/console/src/pages/GetStarted/index.tsx

```diff
@@ -106,6 +110,7 @@ function GetStarted() {
         <div className={styles.title}>{t('get_started.title')}</div>
         <div className={styles.subtitle}>{t('get_started.subtitle')}</div>
       </div>
+      {isDevFeaturesEnabled && isCloud && isDevTenant && <DevelopmentTenantNotification />}
```

Comment:
nit: I think `isDevTenant` should also cover the `isCloud` as it is a cloud only feature